openlibrary/mocks/mock_infobase.py

```diff
@@ -106,6 +106,11 @@ def quicksave(self, key, type="/type/object", **kw):
         }
         query.update(kw)
         self.save(query)
+
+        import logging
+        logger = logging.getLogger(__name__)
+        logger.error("quicksave(%s) query=%s -> %s" % (key, query, self.get(key)))
```

Comment:
Why logging an error (or anything) here?

Reply:
I was looking to see why pytests were failing.  Reverted.